open_schools_platform/organization_management/circles/models.py

```diff
@@ -22,7 +22,7 @@ def create_circle(self, name: str, organization: Organization):
 class Circle(BaseModel):
     id = models.UUIDField(primary_key=True, default=uuid.uuid4)
     name = models.CharField(max_length=200)
-    organization = models.ForeignKey(Organization, related_name="circles", on_delete=models.CASCADE)
+    organization = models.ForeignKey(Organization, null=True, related_name="circles", on_delete=models.CASCADE)
```

Comment:
can you explain this change?

Reply:
I needed it for providing organization field in list_filter I guess, but now I will remove null=True, because we don't need to provide this field in list_filter anymore.